README.md

```diff
@@ -118,6 +118,14 @@ For example, to quickly configure CORS without providing a server.yaml file, it'
 docker run -e JAVA_OPTIONS="-Dinfinispan.cors.enableAll=https://host.domain:port" infinispan/server
 ```
 
+#### Deploying artifacts to the server lib directory
+Deploy artifacts to the server lib directory using the `SERVER_LIBS` env variable.
+For example, to add the PostgreSQL JDBC driver to the server:
+
+```bash
+docker run -e SERVER_LIBS="org.postgresql:postgresql:42.3.1" infinispan/server
```

Comment:
This results in an error for me:

```
Could not download 'org/postgresql/postgresql/42.3.1/postgresql-42.3.1' from 'https://repo1.maven.org/maven2/' repository
IllegalArgumentException: Artifac 'org.postgresql:postgresql:42.3.1' not found

```

Reply:
Fixed with the latest commit from https://github.com/infinispan/infinispan/pull/9676